File: src/main/resources/output/IFC4X3_python/Entities/IfcRepresentationContext/IfcRepresentationContext.py
class IfcRepresentationContext:
    def __init__(self):
        self.ContextIdentifier: IfcLabel = None
        self.ContextType: IfcLabel = None

    # === EXTENDED BY ===
    # IfcGeometricRepresentationContext

    # === INVERSE CLAUSES ===
    # RepresentationsInContext : SET [0:?] OF IfcRepresentation FOR ContextOfItems

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRepresentationContext/IfcRepresentationContext.vb
Public Class IfcRepresentationContext
    Public ContextIdentifier As IfcLabel
    Public ContextType As IfcLabel

    ' === EXTENDED BY ===
    ' IfcGeometricRepresentationContext

    ' === INVERSE CLAUSES ===
    ' RepresentationsInContext : SET [0:?] OF IfcRepresentation FOR ContextOfItems
End Class
